### DISCUSSION TITLE
Lucy and the license to use

### DISCUSSION THREAD

**Discussion by kamikyoku** [gleam-lang/gleam#2679] [Ideas & suggestions]
(For context, to re-introduce myself, I made the current Lucy, the pink starfish. And yes, I've come back after the community discussed about the license to give to the mascot)

[this is going to be long]

I think that anyone is free to come up with their own takes on Lucy. I don't know that much about "professional" artistic licenses (something like the Creative Commons license, which I'm a bit familiar with, but never used it in any of my work so far). But from what I know, "independent artists" (as in the ones found on Twitter or a lot of sites for artists, doing commissions and projects for a living) don't work with strict concepts of licenses that much. It's more common to ask the artist's permission if one wants to, say, make fanart out of their work or reuse their work in another art, than to apply a license which describes some "do"s and "don't"s to every single of their work. 

(If provided a comment section, there's going to be questions along the lines of "can i use this?" or "can i draw this?" with the "i'll give you credit" at the end of such questions)

Sometimes, an artist puts out an agreement (in the commission section of their website, if they have one) on how they'd like to handle commission art they've done, like whether one could post their commissioned art anywhere. I found out this isn't common knowledge in any programming-related community, so I'd like to show a little bit of how artists would work.

As for Lucy's license, I don't know whether it's fine to create another "specialized" license for Lucy (which can go into specific details and cases on how Lucy can be handled in other works), or to use an existing license. As the one who made Lucy, I'm more into "specialized" licenses but that's up to the community and the contributors to decide. Thank you so much if anyone's taken time to go through this block of text, the longest a suggestion could be given.

**Comment by lpil** [comment]
Hello! I was thinking it'd be good to have a look around and see what other projects do for their artwork.

I like the idea of encouraging people to use Lucy for things. I don't especially feel the need to be protective of the art, but maybe I'm being naive.

**Reply by kamikyoku** [reply]
(Sorry I haven't been able to reply all those mails with the packed schedule I'll have this March)

> I was thinking it'd be good to have a look around and see what other projects do for their artwork.

Most projects' artwork license would fall into two categories: artworks that don't allow any usage outside the main project, and the ones that do.

> I don't especially feel the need to be protective of the art, but maybe I'm being naive.

No need to worry about that. I don't want a second copy of Lucy to exist outside of Gleam for fair reasons, but I don't want to restrict anyone in the community to create stuff out of Lucy (this could happen if the license isn't clear about specific cases). Besides, it seems with the packed schedule I may not have much time for Lucy. 

(I'm planning a Q&A about Lucy, but that's for another time...)
